versioned_docs/version-2.6/pages-for-subheaders/manage-clusters.md

```diff
@@ -6,7 +6,6 @@ After you provision a cluster in Rancher, you can begin using powerful Kubernete
 
 This page covers the following topics:
 
-- [Switching between clusters](#switching-between-clusters)
 - [Managing clusters in Rancher](#managing-clusters-in-rancher)
 - [Configuring tools](#configuring-tools)
```

Comment:
Similar to comment on other version of page

Reply:
Removed in 1c0ef1d5c16da08fd39bc39fee15983620b4d475.